src/calendar/calendar.api.json

```diff
@@ -92,9 +102,9 @@
       "description": "Whether to disable days by default"
     },
     {
-      "name": "allowSelectionOutOfRange",
-      "type": "boolean",
-      "description": "Whether to allow selection of dates before minDate or after maxDate"
+      "name": "disabledByWeekDays",
+      "type": "number[]",
+      "description": "Disable dates by days of the week (Sunday=0)"
```

Comment:
Same here. If for example, we set `firstDay` to be Monday then passing `disabledByWeekDays={[1]}` disables Tuesday.

Reply:
No. No matter what is the first day, if you want to disable Mondays you should pass [1]

Reply:
This is what happens if (Tuesday is disabled)
```
firstDay={1}
disabledDaysIndexes={[1]}
```
![image](https://github.com/user-attachments/assets/c56a748a-01ad-40bb-b148-9d6b7728561d)